pkg/controllers/workapplier/controller_integration_test.go

```diff
@@ -286,7 +286,7 @@ func regularJobObjectAppliedActual(nsName, jobName string, appliedWorkOwnerRef *
 	return func() error {
 		// Retrieve the Job object.
 		gotJob := &batchv1.Job{}
-		if err := memberClient.Get(ctx, client.ObjectKey{Namespace: nsName, Name: jobName}, gotJob); err != nil {
+		if err := memberClient1.Get(ctx, client.ObjectKey{Namespace: nsName, Name: jobName}, gotJob); err != nil {
```

Comment:
how did it compile?

Reply:
It doesn’t, that’s the part that puzzles me as well. I found it failing on the main branch hence this PR 🙈 I will turn on the must be up to date to merge option 😖